src/sql/workbench/services/connection/browser/connectionDialogService.ts

```diff
@@ -451,6 +477,7 @@ export class ConnectionDialogService implements IConnectionDialogService {
 			this._connectionDialog.onConnect((profile) => {
 				this.handleOnConnect(this._connectionDialog.newConnectionParams, profile as IConnectionProfile);
 			});
+			// what is input value?
```

Comment:
not sure the context for this comment?

Reply:
Remove:  Reminder to self to check value.